### ISSUE TITLE
Add compatibility for cellranger-arc-2.0.0

### ISSUE THREAD

**Issue by andrea-de-micheli** [MultiQC/MultiQC#1859]
### Name of the tool

CellRanger ARC

### Tool homepage

https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/what-is-cell-ranger-arc

### Tool description

Add compatibility for reports generated by the cellranger-arc-2.0.0 pipeline

### Tool output

[multiome_web_summary.html.zip](https://github.com/ewels/MultiQC/files/10665298/multiome_web_summary.html.zip)

### Log filename pattern

web_summary.html

### Data suitable for MultiQC plot(s)

Similar to #751

**Comment by LucaTucciarone** [comment]
news on this?

**Comment by ewels** [comment]
There are a [lot of](https://github.com/ewels/MultiQC/issues?q=is%3Aopen+is%3Aissue+label%3A%22module%3A+new%22) module requests. Pull requests are welcome 😊 

👉🏻  See [docs](https://multiqc.info/docs/#coding-with-multiqc) on writing new modules.

**Comment by chris-rands** [comment]
Seems done via https://github.com/MultiQC/MultiQC/pull/2925 - suggest to close the issue

**Comment by ewels** [comment]
Yup, done in #2925 as you say. Newer output versions (v2.2) are tracked in #3609. Thanks all!
